docs/descriptor.rst

```diff
@@ -239,20 +241,20 @@ section where you specify package names and repositories to be used.
 
 Packages are defined in the ``install`` subsection.
 
-To be able to define private repositories that you want to use at build time
-we have the ``repositories`` subsection. You can define repository `keys`
-that should be enabled. In image descriptor you only define what to use.
-Actual definition for these repositories takes place in :ref:`Cekit configuration file <configuration_repositories>`.
+Repositories
+^^^^^^^^^^^^^
+Cekit uses all repositories configured inside the image. You can also specify additional
+repositories inside the repositories subsection. Its schema follows these rules:
 
-.. code::
+* **name** - name of the repository object - used mainly for overrides and to reference in  ``~/cekit/config``
```

Comment:
Is name optional? If yes, can be the value of `repository` used for name in it's absence?